### PR TITLE
Initial commit of WCS-related schemas

### PR DESCRIPTION
**Pull request by mcara**

### CODE REVIEW

**Comment by mcara** [comment]
CC: @nden @eslavich @perrygreenfield

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    pip install asdf-wcs-schemas
+
+To install the latest code from the repository
+
+    pip install git+https://github.com/spacetelescope/asdf-wcs-schemas.git
+
+To use the library
+
+    >>> import asdf_wcs_schemas
```
Comment (nden) [inline README.md:19]: Is there ever a need to import schemas?

File: src/asdf_wcs_schemas/_version.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# coding: utf-8
+# file generated by setuptools_scm
+# don't change, don't track in version control
+version = '0.1.dev1+g7d4f13e.d20210715'
+version_tuple = (0, 1, 'dev1+g7d4f13e', 'd20210715')
```
Comment (nden) [inline src/asdf_wcs_schemas/_version.py:5]: This is a generated file, remove from repoitory

**Comment by nden** [comment]
Please remove the STScI logo.

File: src/asdf_wcs_schemas/_version.py (hunk 2 above)
Comment (eslavich) [reply src/asdf_wcs_schemas/_version.py:5]: This repo could use the standard Python gitignore:

https://github.com/github/gitignore/blob/master/Python.gitignore

with this _version.py file included.

**Review by eslavich — COMMENTED** [review]
I tested with both editable and non-editable installs and the schemas and manifest are properly available to asdf in both cases.

**Comment by perrygreenfield** [comment]
To make reviewing simpler, could you highlight the files that are actually different or new? I assume most of these are just relocated from gwcs or elsewhere (and a summary of that somewhere would be good).

**Comment by mcara** [comment]
@perrygreenfield Only schemas are carried over from `gwcs` unchanged. Except for schemas (not including manifest) everything else should be reviewed (even if some code is inspired by similar work on `rad` and `jwst.transforms`).

File: README.md (hunk 1 above)
Comment (mcara) [reply README.md:19]: This `readme` is similar to https://github.com/spacetelescope/rad/blob/main/README.md

**Comment by mcara** [comment]
@perrygreenfield Only schemas are carried over from `gwcs` unchanged. Except for schemas (not including manifest) everything else should be reviewed (even if some code is inspired by similar work on `rad` and `jwst.transforms`).

In other words, ignore `resources/schemas/stsci.edu/gwcs/*.yaml`

**Review by perrygreenfield — COMMENTED** [review]
Aside from wording issues looks good to me.

**Comment by nden** [comment]
Looks good to me. Unless there are major issues we can perhaps merge.

**Review by nden — APPROVED** [review]

**Comment by jdavies-st** [comment]
Merging so we can carry out further testing.

**Comment by mcara** [comment]
Tests are failing because https://github.com/asdf-format/asdf-wcs-schemas was not yet released.

**Pull request merged**